### PR TITLE
Added portal profile filter based on switch port couple

### PR DESCRIPTION
**Pull request by fdurand**
New portal filter like Switch-ifindex

### CODE REVIEW

File: NEWS.asciidoc [hunk 1]
```diff
@@ -17,6 +17,7 @@ Version x.y.x released on yyyy-mm-dd
 New Features
 ++++++++++++
 * pfmon can now run tasks on different intervals
+* added portal profile filter based on switch-port couple
```
Comment (extrafu) [inline NEWS.asciidoc:24]: shouldn't we use "pair" here instead of couple? same everywhere below.

Comment (louismunro) [inline NEWS.asciidoc:24]: Tuple?

Comment (extrafu) [inline NEWS.asciidoc:24]: That would work too - I just had in mind "key-value pair" which is common in programming. Couple is common is procreating ;-)

Comment (louismunro) [inline NEWS.asciidoc:24]: I was only half serious.
Few people would know what I refer to. Pair is more intuitive.

**Comment by julsemaan** [comment]
Can you rebase so we merge in devel. Thanks !

**Pull request merged**